.github/workflows/tests.yml

```diff
@@ -26,14 +31,7 @@ env:
 jobs:
   test:
     if: |
-      github.event_name == 'workflow_dispatch' ||
-      (
-        github.repository == 'matplotlib/matplotlib' &&
-        !contains(github.event.head_commit.message, '[ci skip]') &&
-        !contains(github.event.head_commit.message, '[skip ci]') &&
-        !contains(github.event.head_commit.message, '[skip github]') &&
-        !contains(github.event.head_commit.message, '[ci doc]')
```

Comment:
`[ci doc]` should still be useful if you make a docstring only change.  `[skip github]` is specific to these tests, whereas GitHub’s `[skip actions]` will skip all the GitHub actions.  I don’t know whether that difference is important enough to keep `[skip github]`.